src/main/java/uk/gov/ch/model/update/trusts/IndividualTrusteeData.java

```diff
@@ -1,55 +1,157 @@
 package uk.gov.ch.model.update.trusts;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-
+import java.util.Objects;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Id;
-import java.util.Objects;
+import javax.persistence.Transient;
+import uk.gov.companieshouse.api.model.common.Address;
 
 @Entity
 public class IndividualTrusteeData {
+
     @Id
-    @Column(name = "trustee_id")
+    @Column(name = "TRUSTEE_ID")
     @JsonProperty("trusteeId")
     private String trusteeId;
 
-    @Column(name = "trustee_forename_1")
+    @Column(name = "TRUSTEE_FORENAME_1")
     @JsonProperty("trusteeForename1")
     private String trusteeForename1;
 
-    @Column(name = "trustee_forename_2")
+    @Column(name = "TRUSTEE_FORENAME_2")
     @JsonProperty("trusteeForename2")
     private String trusteeForename2;
 
-    @Column(name = "trustee_surname")
+    @Column(name = "TRUSTEE_SURNAME")
     @JsonProperty("trusteeSurname")
     private String trusteeSurname;
 
-    @Column(name = "date_of_birth")
+    @Column(name = "DATE_OF_BIRTH")
     @JsonProperty("dateOfBirth")
     private String dateOfBirth;
 
-    @Column(name = "nationality")
+    @Column(name = "NATIONALITY")
     @JsonProperty("nationality")
     private String nationality;
 
-    @Column(name = "corporate_ind")
+    @Column(name = "CORPORATE_IND")
     @JsonProperty("corporateIndicator")
     private String corporateIndicator;
 
-    @Column(name = "trustee_type_id")
+    @Column(name = "TRUSTEE_TYPE_ID")
     @JsonProperty("trusteeTypeId")
     private String trusteeTypeId;
 
-    @Column(name = "appointment_date")
+    @Column(name = "APPOINTMENT_DATE")
     @JsonProperty("appointmentDate")
     private String appointmentDate;
 
-    @Column(name = "ceased_date")
+    @Column(name = "CEASED_DATE")
     @JsonProperty("ceasedDate")
     private String ceasedDate;
 
+    @Column(name = "SERVICE_ADDRESS_HOUSE_NAME_NUMBER")
```

Comment:
we had a problem with Oracle DB config for long names. Fingers crossed ...

Reply:
Doesn't seem to have caused an issue when testing locally

Do you know what the limit is?

Reply:
Good point, maybe air on the said of caution and reduce the size just to be safe.. similar to : https://github.com/companieshouse/oracle-query-api/pull/167/files